site/index.md

```diff
@@ -9,10 +9,53 @@ projects/index
 
 Unidata AMS 2021 Python Workshop
 ================================
 
-![under construction](https://images2.minutemediacdn.com/image/upload/c_fit,f_auto,fl_lossy,q_auto,w_728/v1555999902/shape/mentalfloss/under_construction1_0.gif?itok=Pn9g_wu6)
+**Students!:**
+* Are you looking to make the transition to Python but unsure of where to start? 
+* Do you already know Python but want to see atmospheric science specific applications? 
+* Are you looking for data? 
+
+If so, then please join us for a *virtual* hands-on AMS Student Conference Python Workshop where beginners and experts alike will learn skills that enhance their ability to find, analyze, and explore data. All the workshop resources will be in the cloud, so no specialized local software installations are necessary. All you need is a laptop or tablet (a keyboard may be helpful) and a GitHub ID to participate.
+
+```{note}
+**No prior experience** with GitHub or Python is necessary to take part in this workshop!
+```
+
+--- 
+
+**To join us**: begin by [**registering**](https://docs.google.com/forms/d/e/1FAIpQLSdJAC7Zowt_2-Eci83_HYyluHEZCG80RV2o_M1eSPHROcRt2A/viewform) for the workshop. 
+* We'll need your GitHub user ID in order to provision the cloud resources you'll use for the workshop. 
+* If you don't have a GitHub ID yet, don't worry — they're easy to create and the accounts are free. Just click [here](https://github.com/join) to create one.
+
+```{attention}
+Please [register](https://docs.google.com/forms/d/e/1FAIpQLSdJAC7Zowt_2-Eci83_HYyluHEZCG80RV2o_M1eSPHROcRt2A/viewform) by **January 7, 2021**, so we can get everything set up.
+```
+
+--- 
+
+Things will kick off with **two identical orientation sessions** during the AMS Student Conference:
+
+* Session A: Sunday, January 10, 2021, 1:00 PM - 1:30 PM EST
+* Session B: Sunday, January 10, 2021, 1:30 PM - 2:00 PM EST
+
+  ```{tip}
+  Pick one session to attend; they'll both present the same information.
+  ```
+
+---
+
+After the orientation, you'll choose a **project** to work on over the course of the week.  Check out a list of example projects that have been posted [**here**](projects/index.md).
+
+<!-- For some reason, sphynx or our theme does not like linking to a subsection in another page: ie. (agenda.md#post-ams), so I am using the active webpage link instead to link to the subsection for the office hours below. -->
+Remote **follow-up help** will be available.  Take a look at the [Asynchronous Workshop section of our **agenda**](https://unidata.github.io/pyaos-ams-2021/agenda.html#asynchronous-workshop-br-span-class-subhead-throughout-the-week-of-ams-br-january-10th-january-14th-2021-span) to learn about how and when to attend our "office hours”. Workshop sessions and office hours will be staffed by an elite team of student volunteers who have signed up to help show you how using Python can benefit your science.
```

Comment:
According to the [MyST parser docs](https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#auto-generated-header-anchors), something like this should work:

~~~
 [Asynchronous Workshop section of our **agenda**](./agenda.md#asynchronous-workshop-br-span-class-subhead-throughout-the-week-of-ams-br-january-10th-january-14th-2021-span)
~~~

but I think the `./` part is key. Did you try it with the leading `./`? We also might need to enable ` myst_heading_anchors=2` in `conf.py`, as suggested in the docs...but the anchors seem to be generated in our docs as-is, so that might not be necessary with the theme we are using.

Reply:
Looks like it's a no-go, so we'll leave it as-is. 👍

Reply:
I was testing with the simplest section (#post-ams) for test purposes to start with.  It didn't like any iteration of: 
`agenda.md#post-ams
/agenda.md#post-ams
./agenda.md#post-ams`
unfortunately.